challenges/identification/I3_benchmark/I3_specification.md

```diff
@@ -0,0 +1,113 @@
+
+
+## Outputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | JSON | [Specification][spec-json] | [Link][out1] |
+  | 2 | JSON | [Specification][spec-json] | [Link][out2] |
+  
+### Additional info outputs
+
+#### Format 1
+  
+  The following table lists the attribute names, value types and units, and a
+description of each attribute-value pair:
+  
+  | Attribute | Type | Unit | Description |
+  | --- | --- | --- | --- |
+  | `Precision_50nt` | `float` | N/A | Precision of PAS identification compared with PolyASite database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 50 nt |
+  | `Precision_100nt` | `float` | N/A | Precision of PAS identification compared with PolyASite database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 100 nt |
+  | `Precision_200nt` | `float` | N/A | Precision of PAS identification compared with PolyASite database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 200 nt |
+
+#### Format 2
+  
+  The following table lists the attribute names, value types and units, and a
+description of each attribute-value pair:
+  
+  | Attribute | Type | Unit | Description |
+  | --- | --- | --- | --- |
+  | `Precision_50nt` | `float` | N/A | Precision of PAS identification compared with PolyA_DB database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 50 nt |
+  | `Precision_100nt` | `float` | N/A | Precision of PAS identification compared with PolyA_DB database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 100 nt |
+  | `Precision_200nt` | `float` | N/A | Precision of PAS identification compared with PolyA_DB database; Precision = (TP/(TP+FP)); maximum distance between identified PAS and PAS in database: 200 nt |
+  
+## Metrics
+  
+  | # | Description | Unit | Compute from | Transformations | Type after transformations | Additional comments |
+  | --- | --- | --- | --- | --- | --- | --- |
+  | 1 | Precision_50nt_PolyASite | N/A | Output 1 | Read file, parse JSON and extract attribute `Precision_50nt` | `vector` | N/A |
+  | 2 | Precision_100nt_PolyASite | N/A | Output 1 | Read file, parse JSON and extract attribute `Precision_100nt` | `vector` | N/A |
+  | 3 | Precision_200nt_PolyASite | N/A | Output 1 | Read file, parse JSON and extract attribute `Precision_200nt` | `vector` | N/A |
+  | 4 | Precision_50nt_PolyAdb | N/A | Output 2 | Read file, parse JSON and extract attribute `Precision_50nt` | `vector` | N/A |
+  | 5 | Precision_100nt_PolyAdb | N/A | Output 2 | Read file, parse JSON and extract attribute `Precision_100nt` | `vector` | N/A |
+  | 6 | Precision_200nt_PolyAdb | N/A | Output 2 | Read file, parse JSON and extract attribute `Precision_200nt` | `vector` | N/A |
+  
+### Additional info metrics
+  
+  N/A
+
+[//]: # (References)
+  
+  [in1]: ./example_files/input1.bed
```

Comment:
Can we add versions of the input files? `input1.bed` can be same as before for I2. If the inputs stay as polyASite and PolyA_DB formats, we can just dump a `head -n10` or so for them.

Reply:
@daneckaw I sent a PR to your branch with these.